src/open_inwoner/accounts/views/inbox.py

```diff
@@ -105,7 +105,7 @@ def get_other_user(self, conversations: dict) -> Optional[User]:
 
         return get_object_or_404(User, uuid=other_user_uuid)
 
-    def get_messages(self, other_user: User) -> MessageQuerySet:
```

Comment:
I'm not sure why `List` was added here, it just returns a queryset right?

Reply:
Slicing a queryset results in another queryset, but slicing with step (`queryset[1:10:2]`) results in a list. Who would have thought this might lead to confusion and bugs :)

Reply:
But it would then be a list of messages, not a list of querysets right?